emu/src/emu_display.c

```diff
@@ -642,9 +642,9 @@ void setLeds(led_t* leds, uint8_t numLeds)
 	pthread_mutex_lock(&displayMutex);
     for(int i = 0; i < numLeds; i++)
     {
-        rdLeds[i].r = leds[i].r >> ledBrightness;
-        rdLeds[i].g = leds[i].g >> ledBrightness;
-        rdLeds[i].b = leds[i].b >> ledBrightness;
+        rdLeds[i].r = leds[i].r;// >> ledBrightness;
```

Comment:
Why did you remove emulated LED brightness control? If you set it to max brightness, it'd be equivalent. Please revert this/

Reply:
Fixed.
I had set it this way because the LEDs were barely visible in my emulator.